### DISCUSSION TITLE
Config File reset for my car

### DISCUSSION THREAD

**Discussion by fraan15** [cedricp/ddt4all#1001] [Q&A]
Hello, a while ago I played with ddt4all and modified options that I should not have modified and I did not make a backup, now I find that my rlink works incorrectly and I do not know what parameters I have modified, I have read that it is possible to generate a file with the default configuration of my vehicle through the VIN, the version I had my rlink with was v7, my VIN is VF1RFB00260616209, I would be very grateful if someone could tell me how to generate these files or generate them for me, thank you very much.

**Comment by KarelSvo** [comment]
https://github.com/cedricp/ddt4all/discussions/967

**Reply by fraan15** [reply]
Thank you very much, I have been reading the thread and I have generated my zip with the telegram bot, now I understand that by connecting with odb and pyren I have to run a macro with that zip so that it inserts this data into rlink?

**Comment by fraan15** [comment]
Now i need extract zip files in macros folder and press macro button on pyren?

**Reply by KarelSvo** [reply]
The .cmd files go into the macro folder.
